### PR TITLE
Automate chromium update

### LINKED ISSUE #1119: Update chromium dependency versions automatically
Write a GitHub action to check for the latest release of the chromium browser so that Taiko's chromium dependency is always upto date.

This github action must raise a PR with the update in versions at 

https://github.com/getgauge/taiko/blob/27d6856a673eb8aa999259e507721bf00cc58f33/package.json#L38-L41

### PR DESCRIPTION
**Pull request by NivedhaSenthil**
Fixes #1119

### CODE REVIEW

File: .github/workflows/update_chromium.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                       libxt6
+          npm install
+          export VERSION="$(bin/taiko.js getVersion.js --no-log)"
+          npx json -I -f package.json -e "this.taiko.chromium_version = '$VERSION'"
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v2
+        with:
+          token: ${{ secrets.GAUGEBOT_GITHUB_TOKEN }}
+          branch: ${{format('{{{0}{1}{2}}}', 'update-chromium', github.run_id, github.run_number)}}
+          commit-message: "update chromium revision"
```
Comment (zabil) [inline .github/workflows/update_chromium.yml]: Can this be `Update chromium to revision [number]` ?

Comment (NivedhaSenthil) [reply .github/workflows/update_chromium.yml]: Done 👍

File: .github/workflows/update_chromium.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                       libxt6
+          npm install
+          export VERSION="$(bin/taiko.js getVersion.js --no-log)"
+          npx json -I -f package.json -e "this.taiko.chromium_version = '$VERSION'"
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v2
+        with:
+          token: ${{ secrets.GAUGEBOT_GITHUB_TOKEN }}
+          branch: ${{format('{0}{1}', 'update-chromium',env.REVISION)}}
+          commit-message: ${{format('{0}{1}','update chromium to revision ',env.REVISION)}}
```
Comment (zabil) [inline .github/workflows/update_chromium.yml]:
```suggestion
          commit-message: ${{format('{0}{1}','Update chromium to revision ', env.REVISION)}}
```

File: .github/workflows/update_chromium.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+          title: ${{format('{0}{1}','update chromium to revision ',env.REVISION)}}
```
Comment (zabil) [inline .github/workflows/update_chromium.yml]:
```suggestion
          title: ${{format('{0}{1}','Update chromium to revision ', env.REVISION)}}
```

**Pull request merged**